packages/wonder-blocks-modal/components/modal-backdrop.js

```diff
@@ -42,7 +42,10 @@ export default class ModalBackdrop extends React.Component<Props> {
         if (!lastButton) {
             return;
         }
-        lastButton.focus();
+        // wait for styles to applied
+        setTimeout(() => {
+            lastButton.focus();
+        }, 0);
```

Comment:
I hate the `setTimeout` hack in general but in specific, I think it's made for situations like this. :)

Shame Aphrodite doesn't have some sort of "I've applied styles" callback.

Reply:
That'd be cool.  I'm not sure how to wire it up so that tests would have access to it.